packages/dds/tree/src/feature-libraries/chunked-forest/codec/chunkDecoding.ts

```diff
@@ -228,6 +234,33 @@ export class InlineArrayDecoder implements ChunkDecoder {
 	}
 }
 
+/**
+ * Decoder for {@link EncodedIncrementalShape}s.
+ */
+export class IncrementalFieldDecoder implements ChunkDecoder {
```

Comment:
Since there is no reason this could not be used to decode part of a field, I don't think we should call it an `IncrementalFieldDecoder`

`IncrementalChunkDecoder` would better describe its true capabilities.

Future optimization will likely want to leverage this when encoding very long sequences as trees ( a top level of incrementla chunks for the field, but inside them more incremental chunks, which do not correspond to a whole field)

Reply:
Updated.